common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -2,11 +2,16 @@
 """
 End-to-end tests for the main LMS Dashboard (aka, Student Dashboard).
 """
+import datetime
+
 from ..helpers import UniqueCourseTest
 from ...fixtures.course import CourseFixture
 from ...pages.lms.auto_auth import AutoAuthPage
 from ...pages.lms.dashboard import DashboardPage
 
+DEFAULT_SHORT_DATE_FORMAT = "%b %d, %Y"
+DEFAULT_DAY_AND_TIME_FORMAT = "%A at %-I%P"
```

Comment:
These constants should be imported from `common/djangoapps/util/date_utils.py` instead of declaring here.